website/docs/migration.md

```diff
@@ -0,0 +1,213 @@
+---
+sidebar_position: 9
+---
+
+# Migrating from MongoDB to FerretDB
+
+To ensure a smooth and successful migration from MongoDB, we offer several methods through which you can test your application with FerretDB.
+
+## Operation modes
+
+We offer multiple operation modes which help facilitate the testing of your application by enabling FerretDB to act as a proxy.
+For more details, refer to the [official documentation](./configuration/operation-modes.md).
+
+### Manual and automated testing with `diff-normal` mode
+
+For details on how to install FerretDB, refer to the [quickstart guide](../quickstart-guide/).
+
+You can manually test your application or use integration tests, among other methods.
+Afterward, you can inspect the differential output for errors or inconsistencies between responses that require your attention.
+
+As an example, let us say that your application performs some complex query and you'd like to test it in `diff-normal` mode.
+You would do the following:
+
+1. Start FerretDB in `diff-normal` mode.
+   This can be achieved by using the `--mode` [flag](./configuration/flags.md) or by setting the `FERRETDB_MODE` environment variable.
```

Comment:
```suggestion
   This can be achieved by using the `--mode` [flag](configuration/flags.md) or by setting the `FERRETDB_MODE` environment variable.
```